biggraphite/plugins/graphite.py

```diff
@@ -142,8 +142,9 @@ def accessor(self):
         """Return an accessor."""
         if not self._accessor:
             from django.conf import settings as django_settings
-            self._accessor = graphite_utils.accessor_from_settings(django_settings)
-            self._accessor.connect()
+            accessor = graphite_utils.accessor_from_settings(django_settings)
+            accessor.connect()
```

Comment:
Could you comment this line explaining that the connect on the accessor might error out and therefore we never return an unconnected one here?